### ISSUE TITLE
Rename file option missing

### ISSUE THREAD

**Issue by Urgen-Dorjee** [MicrosoftDocs/visualstudio-docs#3356]
When you create a new class project, it comes with default file name called Class1.cs. There used to be an option to rename this to file<Quick Action> menu, but now it does not show up. when you rename Class1.cs(whatever) to Mynewclass.cs in the editor, it does not rename the file. Hence I have to manually rename the file. It was working fine in VS2017 but VS2019 it is not. I browsed through Tools>options but could not find any option related to this feature. May be you can help!!

**Comment by gewarren** [comment]
@Urgen-Dorjee Are you referring to the dialog box that pops up in VS 2017 when you rename a file in Solution Explorer?

![image](https://user-images.githubusercontent.com/24882762/58825773-068e1500-85f4-11e9-8056-983ac445bd37.png)

@kendrahavens Do you have any insight into why that functionality was removed from VS 2019?

@Urgen-Dorjee There is a "quick action" that you can invoke to rename the type to match the file name. Does that help? https://docs.microsoft.com/visualstudio/ide/reference/sync-type-and-file?view=vs-2019

**Comment by Urgen-Dorjee** [comment]
@gewarren, Yes this dialog box should show up, when you rename a file to replace related class name (vice-versa). This was working fine with my VS2017 IDE. I already checked this [Link](https://docs.microsoft.com/en-us/visualstudio/ide/reference/sync-type-and-file?view=vs-2019)
as you have pointed here but could not find any help!. Quick Action's icon does not have any choice to rename file's name rather rename it back to original one or generate a constructor options available. Therefore I was wondering whether this feature has been removed from VS2019 or m I missing some settings? 

Thanks for your help.
